src/e2e/puppeteer/__tests__/sidebar.ts

```diff
@@ -0,0 +1,42 @@
+import path from 'path'
+import configureSnapshots from '../configureSnapshots'
+import click from '../helpers/click'
+import press from '../helpers/press'
+import screenshot from '../helpers/screenshot'
+import setTheme from '../helpers/setTheme'
+import type from '../helpers/type'
+
+expect.extend({
+  toMatchImageSnapshot: configureSnapshots({ fileName: path.basename(__filename).replace('.ts', '') }),
+})
+
+vi.setConfig({ testTimeout: 20000, hookTimeout: 20000 })
+
+/** Open sidebar and wait for it to slide all the way open. */
+const fullyOpenSidebar = async () => {
+  await click('[aria-label=menu]')
+  await new Promise(resolve => setTimeout(resolve, 2000))
+}
+
+describe('sidebar', () => {
+  it('empty sidebar on dark theme', async () => {
+    await fullyOpenSidebar()
+
+    expect(await screenshot()).toMatchImageSnapshot({ customSnapshotIdentifier: 'sidebar-empty' })
```

Comment:
In the sidebar tests, could you hide the toolbar buttons like I did in https://github.com/cybersemics/em/blob/36edb98e159d91721b610db1e1d4e512f886d690/src/e2e/puppeteer/__tests__/ColorPicker.ts? That will prevent churn when the toolbar changes. Visual regressions of the toolbar in Light mode will get picked up by render-thoughts initial load.

```ts
await hideVisibility('[data-testid="toolbar-icon"]')
```